### ISSUE TITLE
sapnwrfc.dll

### ISSUE THREAD

**Issue by LuigiMortari** [nunomaia/NwRfcNet#50]
Hi.
It's seems that the package requires sapnwrfc.dll.
Correct?

**Comment by nunomaia** [comment]
yes, SAP license do not allow to redistribute sapnwrfc, download it from sap note 2573790
